src/featbin/extract-rows.cc

```diff
@@ -149,7 +149,7 @@ int main(int argc, char *argv[]) {
     KALDI_LOG << "Processed " << num_done << " segments successfully; "
               << "errors on " << num_err;
 
-    return (num_done > 0);
+    return (num_done == 0);
```

Comment:
Yes, that was a bug; that code has not been used much so far.
thanks, but can you please change it to
(num_done > 0 ? 0 : 1)
which is how we normally write it?
returning the truth value causes me to do a double-take when looking at the code.